### ISSUE TITLE
Features should have accessible names for discernability

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#316]
The features are indistinguishable to someone relying on a screen reader as they don't have an accessible name. `<path tabindex="0">` is announced as <q>graphic</q> in Chrome (<q>blank</q> in Firefox):

<table>
<td width="400">
<video src="https://user-images.githubusercontent.com/26493779/106403524-6c691a80-642f-11eb-8bb8-62b633059214.mp4"></video>
</td></table>

_Originally posted by @Malvoz in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/270#issuecomment-770489852_

**Comment by Malvoz** [comment]
I see that the `<path>` element supports a child `<title>` element that provides an accessible name.  If we can come up with a convention for providing a similar accessible name for `<feature>`, we could generate the `<path><title>` from it.

_Originally posted by @prushforth in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/270#issuecomment-770502203_

**Comment by Malvoz** [comment]
Given the extremely generic properties model that `<feature>` has, I wonder if we should *add* a `<title>` element child of `<properties>` as a special case for accessible names purposes similar to svg.  In the polyfill we can map that to <path><title> or what have you. I'll open an issue on this latter discussion in the MapML and UCR document repos.

_Originally posted by @prushforth in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/270#issuecomment-770979801_

**Comment by prushforth** [comment]
I believe that title is a global HTML attribute, so maybe it makes sense to respect / use a `<feature title>` as an accessible name. We don't have to worry about conflicts between a `<title>` and `<feature title>`.

**Comment by Malvoz** [comment]
In that case we should consider `label` as opposed to `title`.

**Comment by Malvoz** [comment]
`label` is element-specific per element's definitions, and more encouraging for authors to use as opposed to `title` which feels more "optional" since it's a global. `title` also implies the native tooltip on hover.

**Comment by prushforth** [comment]
This is resolved by opting for a `<feature><featurecaption>` element for the moment.
